IPython/core/logger.py

```diff
@@ -193,8 +192,8 @@ def log_write(self, data, kind='input'):
             write = self.logfile.write
             if kind=='input':
                 if self.timestamp:
-                    write(str_to_unicode(time.strftime('# %a, %d %b %Y %H:%M:%S\n',
-                                        time.localtime())))
+                    write(str(time.strftime('# %a, %d %b %Y %H:%M:%S\n',
```

Comment:
The `str` here shouldn't be necessary, I think. `str_to_unicode` is a no-op on Python 3, because `str` is unicode.